components/DynamicContent/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,846 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+
+import "../index.scss";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS/Layout/Dynamic content"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="DynamicContent" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <section class="denhaag-dynamic-content denhaag-dynamic-content--grid">
+      <div class="denhaag-dynamic-content__grid container">
+        <article class="denhaag-dynamic-content__card">
+          <img
+            class="denhaag-dynamic-content__card-image denhaag-image__image"
+            srcset="https://via.placeholder.com/340x190/90D40166 480w,
+                https://via.placeholder.com/510x285/90D40166 768w,
+                https://via.placeholder.com/680x380/90D40166 1290w"
+            src="https://via.placeholder.com/340x190/90D40166"
+            alt=""
+            loading="lazy"
+          />
+          <div class="denhaag-dynamic-content__card-content">
+            <div class="denhaag-dynamic-content__card-meta">
```

Comment:
Should'nt 'denhaag-dynamic-content__card-meta' be more descriptive like 'denhaag-dynamic-content__card-caption'?